code/game/objects/structures/safe.dm

```diff
@@ -176,17 +177,17 @@ GLOBAL_LIST_EMPTY(safes)
 			if("Cancel")
 				return
 	else
-		ui_interact(user)
+		tgui_interact(user)
 
-/obj/structure/safe/ui_interact(mob/user, ui_key = "main", datum/nanoui/ui = null, force_open = 1)
-	ui = SSnanoui.try_update_ui(user, src, ui_key, ui, force_open)
+/obj/structure/safe/tgui_interact(mob/user, ui_key = "main", datum/tgui/ui = null, force_open = FALSE, datum/tgui/master_ui = null, datum/tgui_state/state = GLOB.tgui_default_state)
```

Comment:
```suggestion
/obj/structure/safe/tgui_interact(mob/user, ui_key = "main", datum/tgui/ui = null, force_open = FALSE, datum/tgui/master_ui = null, datum/tgui_state/state = GLOB.tgui_physical_state)
```
Safes are physical, not electronic. Use `physical_state` to avoid AIs and borgs using them